marconi-sidechain/README.md

```diff
@@ -195,7 +195,23 @@ $ curl -d '{"jsonrpc": "2.0" , "method": "getUtxosFromAddress" , "params": { "ad
 
 #### getTxsBurningAssetId
 
-Example yet to come.
+```sh
+$ curl -d '{"jsonrpc": "2.0", "method": "getTxsBurningAssetId", "params": {"policyId": "fda1b6b487bee2e7f64ecf24d24b1224342484c0195ee1b7b943db50", "assetName": "4c6f6273746572436f756e746572"}, "id": 1}' -H 'Content-Type: application/json' -X POST http://localhost:3000/json-rpc | jq
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": [
+    {
+      "blockHeaderHash": "834ac116737ed735805864491626aa8eadca3a9c3be86b559c65c92fd3ddd9fa",
+      "quantity": -22411,
+      "redeemer": "80",
+      "redeemerHash": null,
```

Comment:
Should not be null.

Reply:
Hmm, looks like we are not indexing it at all, it's set to Nothing [for all results](https://github.com/input-output-hk/marconi/blob/7cdedd3a4ec05f989b9d001a3676e57924376822/marconi-sidechain/src/Marconi/Sidechain/Api/Query/Indexers/MintBurn.hs#L75).

Reply:
Yeah, we should compute it. Can you add it? Should not be complicated, it's just a call to `hashScriptData` from cardano-api.

Reply:
Will do!

Reply:
Added the hash 497179774, ran the query again to get the hash for the sample f4a652c66.